### DISCUSSION TITLE
Enter setup mode

### DISCUSSION THREAD

**Discussion by willliamchan** [RavenSystem/esp-homekit-devices#1675] [Q&A]
Hi there

I am not very sure why but the 8 times power toggle never work with my two HAA 9.5.1 device, is there any other way to enter setup mode again without re-loading the firmware?

Million thanks!

W

**Comment by tonysprenk** [comment]
https://apps.apple.com/nl/app/haa-home-manager-for-homekit/id1556105121?l=en

**Comment by willliamchan** [comment]
Yes I got the app too and unfortunately the old WiFi no longer exists, any other suggestion please?
________________________________

**Comment by tonysprenk** [comment]
If the WiFi doesn’t exist the device should automatically go into AP mode after reboot. So reboot the device (unplug from power and plug back in) and then connect your phone or computer to the HAA-XXXX network. For future reference: please only use the issue section for faults with the firmware. The discussion section is meant for user questions.

**Comment by willliamchan** [comment]
Gotcha. Sorry about that.

P.S. unfortunately I can’t see the HAA AP either, must be something to do with the device itself.


_____________________________________________

​William Chan


HK +852 9661 8686

UK +44 7898 888686


***@***.***

________________________________

**Comment by tonysprenk** [comment]
Have you tried [emergency setup mode](https://github.com/RavenSystem/esp-homekit-devices/wiki/Setup-Mode#entering-setup-mode) ?

**Comment by willliamchan** [comment]
Yes I did, hotspot appear from network list for about a min but never able to connect.

Restarting the phone and/or using another phone are the same.


_____________________________________________

​William Chan


HK +852 9661 8686

UK +44 7898 888686


***@***.***

________________________________

**Comment by i3laze** [comment]
Actually, even in emergency setup mode  device respects WiFi settings and tries to connect to your WiFi access point.
Either turn off your WiFi AP, or check router's DHCP logs for HAA IP address and try http://**IP**:4567.

**Comment by willliamchan** [comment]
After several failures of joining HAA network and it worked.

Thank you very much!


_____________________________________________

​William Chan


HK +852 9661 8686

UK +44 7898 888686


***@***.***

________________________________

**Reply by fbe28** [reply]
Hey, @willliamchan !

Glad to hear you managed to fix this.. I'm running into the exact same issue with a Sonoff Mini R2. Do you have any insight that could help me, perhaps?

I'm not even seeing the device in Setup Radar in the iOS App anymore.. :/
HAA generated AP shows for a few seconds, nor Windows or MacOS can connect to it.. Tried over 15 times and no luck.

Any help is appreciated. Thanks so much!!

**Comment by willliamchan** [comment]
For me the 8 power on/off not working every time and what I did was just keep trying. Also you will need to be really close to the device to connect (for me I need to keep my phone in 1 meter or less.


_____________________________________________

​William Chan


HK +852 9661 8686

UK +44 7898 888686


***@***.***

________________________________

**Reply by fbe28** [reply]
Yes, makes sense.. It's exactly the same here, seems like it's completely hit or miss, unfortunately. I have been trying with the device like 10cm away from my phone, literally side by side, doesn't seem to make a difference.

After trying several times, sometimes it works, but even if the AP shows up, I can connect to it and nothing happens.

In the app, it shows the device in setup mode probably once out of 15-20 tries using the 8x button press method :/

Something seems to be going wrong with this, I just don't know how to tell if it's the device or the firmware, there is never an error message or anything like that..

**Comment by willliamchan** [comment]
The moment I can re-configure the wifi the next thing I did was update it’s firmware, so from there I can start using the HAA app to manage it.
